test/functional/api/kernel.py

```diff
@@ -0,0 +1,121 @@
+from core.test_run_utils import TestRun
```

Comment:
I wonder if `test-framework/test_utils/os_utils.py` wouldn't be a better place for kernel utils. There is no CAS specific operations here.